trino-hms-hdfs-ranger/tests/trino/5_test_trino_select_alter_perm.sh

```diff
@@ -49,12 +49,12 @@ retryOperationIfNeeded "$abs_path" "alterTrinoTable $TRINO_TABLE $NEW_TRINO_TABL
 # successMsg="INSERT: 1 row"
 # retryOperationIfNeeded "$abs_path" "performTrinoCmd $cmd" "$successMsg" "false"
 
-# TODO: Fix this test
+# FIXME: Comment out this test for Hive 4
 # It fails with the following error: "Query 20240402_105548_00016_g47rj failed: Cannot delete from non-managed Hive table".
 # It should not fail because $TABLE_ANIMALS is created in Hive managed space.
-#echo ""
-#echo "- INFO: Delete from $TABLE_ANIMALS table."
-#echo "- INFO: [alter] should succeed."
-#cmd="delete from hive.default.$TABLE_ANIMALS;"
-#successMsg="DELETE"
-#retryOperationIfNeeded "$abs_path" "performTrinoCmd $cmd" "$successMsg" "false"
+echo ""
```

Comment:
Instead of commenting the test for Hive4, we can choose whether to run it or not, depending on the Hive version.

```suggestion
  if [ "$HIVE_VERSION" != "4" ]; then
    echo ""
    echo "- INFO: Delete from $TABLE_ANIMALS table."
    echo "- INFO: [alter] should succeed."
    cmd="delete from hive.default.$TABLE_ANIMALS;"
    successMsg="DELETE"
    retryOperationIfNeeded "$abs_path" "performTrinoCmd $cmd" "$successMsg" "false"
  fi
```

Because we never check for `3` but we check whether `4` exists or not, the check should be `!= 4` and not `== 3`.

`HIVE_VERSION` will be available here, right?

Reply:
Handy! It works, I've tested it out.